zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -528,6 +528,20 @@ impl<F: FieldExt> From<[F; 11]> for RwRow<F> {
 }
 
 impl Rw {
+    pub fn get_rw_counter(&self) -> usize {
```

Comment:
it seems changing method name to `rw_counter` will make it more consistent with other methods?

Reply:
nice to me,  apply it in [2d483e7](https://github.com/appliedzkp/zkevm-circuits/pull/415/commits/2d483e7dc2f5c95d7c911b2243afc4fb6fe8a0d2) thanks